ui/src/app/what-if/what-if.component.html

```diff
@@ -0,0 +1,45 @@
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
+<div class = "whatif">
+<label id = "workload_title">{{addWorkload}}</label>
+
+<!--name-->
+<div class = "name">
+<form clrForm>
+  <clr-input-container>
+    <label id = "scene_name">Scenario Name :</label>
+    <input type = "text" placeholder="What's your scenario?" name="name" id = "scene_input"[(ngModel)]="scenarioName"/>
+  </clr-input-container>
+</form>
+</div>
+
+<!--profile-->
+<div class = "app_profile">
+  <label id = "profile_title">{{appProfile}} :</label>
+  <div class = "app_entries">
+    <div *ngFor = "let p of profile; let i = index" ><label class = "single_label">{{p.label}}</label><div > <input type="text" [(ngModel)] = "profile_values[i]" name="entry" class = "single_input"/></div><label class = "single_unit">{{p.unit}}</label></div>
+  </div>
+</div>
+
+<!--buttons-->
+<div class = "workload_btn">
+  <button class="btn btn-success" id = "run" (click) = "runScenario()">Run Scenario</button>
+  <button class="btn btn-primary" id = "cancel" routerLink = "/group">Cancel</button>
```

Comment:
Cancel should not be a btn-primary button. Please use some other class like "outline"